demo/app/components/drawer/drawer.component.html

```diff
@@ -92,6 +101,18 @@
           The #1 software development tool used by agile teams.
           Visual regression testing is the ability to cross-reference snapshots of a product, highlighting pixel differences — returning fast feedback on the current state of the screen.
         </div>
+        <div>
+          <img src="https://d3vv6lp55qjaqc.cloudfront.net/items/20322G0V3H2j3n2M2o3l/nice%20laptop%3Ascreen%20elevation.png">
```

Comment:
Thoughts on using something like https://placeholder.com/ instead? This way we aren't paying for the traffic and don't need to worry about these links going away..